### PR TITLE
Fix benchmarking MobileNetV2 on S20 GPU in Linalg on tensor path.

### PR DESCRIPTION
**Pull request by hanhanW**
See https://github.com/google/iree/pull/5282 for more details.

The Linalg on tensors path should work with `-iree-spirv-enable-vectorization`, also added the forgotten flag.

### CODE REVIEW

**Comment by hanhanW** [comment]
Triggered the test in https://buildkite.com/iree/iree-android-benchmark/builds/2227

File: build_tools/mako/configuration.py [hunk 1]
```diff
@@ -176,7 +176,10 @@ def get_s20_default_target_list(skipped_target=None, batch_config=None):
               "--iree-vulkan-target-triple=valhall-g77-unknown-android10",
               "--iree-flow-dispatch-linalg-on-tensors",
               "--iree-codegen-spirv-experimental-linalg-on-tensors",
-              "-iree-flow-inline-constants-max-byte-length=2048"
+              "-iree-flow-inline-constants-max-byte-length=2048",
+              "-iree-spirv-enable-vectorization",
```
Comment (antiagainst) [inline build_tools/mako/configuration.py:180]: Actually this is not needed given https://github.com/google/iree/pull/5280. But good to keep it here for clarity.

File: build_tools/mako/configuration.py [hunk 2]
```diff
@@ -176,7 +176,10 @@ def get_s20_default_target_list(skipped_target=None, batch_config=None):
...
+              # TODO(GH-5330): Remove the flag.
+              "-iree-flow-inline-constants-max-byte-length=16"
```
Comment (antiagainst) [inline build_tools/mako/configuration.py:182]: This is duplicated with L179. We should delete 179.

**Review by antiagainst — CHANGES_REQUESTED** [review]

File: build_tools/mako/configuration.py (hunk 2 above)
Comment (hanhanW) [reply build_tools/mako/configuration.py]: ah, good catch...

File: build_tools/mako/configuration.py (hunk 1 above)
Comment (hanhanW) [reply build_tools/mako/configuration.py]: got you, thanks!

File: build_tools/mako/configuration.py [hunk 3]
```diff
@@ -176,7 +176,9 @@ def get_s20_default_target_list(skipped_target=None, batch_config=None):
               "--iree-vulkan-target-triple=valhall-g77-unknown-android10",
               "--iree-flow-dispatch-linalg-on-tensors",
               "--iree-codegen-spirv-experimental-linalg-on-tensors",
-              "-iree-flow-inline-constants-max-byte-length=2048"
+              # TODO(GH-5330): Replace 16 with 2048 or delete the flag.
```
Comment (antiagainst) [inline build_tools/mako/configuration.py:179]: Not sure 2048 is a good configuration. Wit it we will be putting too many constant values into the kernel. 2048 is a workaround for the descriptor issue. I'd just say "revisit the number or delete the flag" here.

**Review by antiagainst — APPROVED** [review]

**Pull request merged**